pkg/healthcheck/healthcheck.go

```diff
@@ -1370,6 +1383,32 @@ func (hc *HealthChecker) checkCanGet(namespace, group, version, resource string)
 	return hc.checkCanPerformAction("get", namespace, group, version, resource)
 }
 
+func (hc *HealthChecker) checkAPIService(serviceName string) error {
+	apiServiceClient, err := apiregistrationv1client.NewForConfig(hc.kubeAPI.Config)
+	if err != nil {
+		return err
+	}
+
+	apiStatus, err := apiServiceClient.APIServices().Get(serviceName, metav1.GetOptions{})
+	if err != nil && len(apiStatus.Status.Conditions) == 0 {
+		return err
+	}
+
+	available := false
+	var errorMessage string
+	for _, condition := range apiStatus.Status.Conditions {
+		if condition.Status == "True" {
+			available = true
+			break
+		}
+		errorMessage = fmt.Sprintf("%s:%s", condition.Reason, condition.Message)
```

Comment:
Can we keep this `%s: %s`? to make them similar with other errors.

Reply:
Yes!! Please check now